### ISSUE TITLE
'Redo last screenshot' uses wrong window size and position

### ISSUE THREAD

**Issue by dreyTee** [shutter-project/shutter#626]
#### Brief summary of issue
'Redo last screenshot' uses wrong window size and position

#### Steps to reproduce the issue

1. Make needed area screenshot
2. Use 'Redo last screenshot' option
3. check results

#### Error output
No error, just using  not previous screenshot window position and size

#### Extra information, such as Shutter version, display server in use (Xorg or Wayland), operating system and ideas for how to solve:
Shutter ver.: 0.99.2 Rev.1593
OS: 
DISTRIB_DESCRIPTION="Linux Mint 21 Vanessa"
ID_LIKE="ubuntu debian"
UBUNTU_CODENAME=jammy
Xorg

**Comment by Photon89** [comment]
I haven't seen this so far. Do you possibly use a multimonitor setup or HiDPI?

**Comment by dreyTee** [comment]
No, one monitor. The only thing changed from my previous setup is that now I use scaling.
I have a 14" monitor and 2880x1800 resolution, scaling 200%.
In logs I see that shutter thinks I have 1440x900 resolution:
`Using monitor: 0 - 0 - 1440 - 900`

I did few tests and now i see:
- sometimes the size of Selection2 is twice of the Selection1 size. [Log](https://pastebin.com/a9NXhfSS)
- sometimes it reports `Error while taking a screenshot` (if I choose an area bigger then the quarter of the screen or I choose it too close to the bottom or right edge). [Log from console](https://pastebin.com/ZDWHGL9y)

**Comment by Photon89** [comment]
Unfortunately, there is no proper HiDPI (that is, resolution scaling) support yet, related: https://github.com/shutter-project/shutter/issues/326
